### PR TITLE
issue #46 Integrate adobe analytics

### LINKED ISSUE #46: Integrate Analytics
Check with customer which solution they use.

### PR DESCRIPTION
**Pull request by iamitshri**
Fix #46

Test URLs:
- Before: https://main--24life--hlxsites.hlx.page/
- After: https://46-integrate-analytics--24life--hlxsites.hlx.page/

### CODE REVIEW

File: 24life/scripts/delayed.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+  const isProd = window.location.hostname === 'https://www.24hourfitness.com/';
+  const launchProd = 'https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN899f16b777754991924d45661d0c60bb.min.js';
+  const launchStaging = 'https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN1720538a14c5448383dbc3a400a4f55a-staging.min.js';
+  await loadScript(isProd ? launchProd : launchStaging, {
+    type: 'text/javascript',
+    charset: 'UTF-8',
+    async: true,
+  });
+}
+
```
Comment (iamitshri) [inline 24life/scripts/delayed.js:19]: Dev: `"https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN95eda61e2ae9436886ce63d7d3dcb671-development.min.js"`

Staging:` "https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN1720538a14c5448383dbc3a400a4f55a-staging.min.js"`


Production: `"https://assets.adobedtm.com/57921990a5e5/01b20556a634/launch-EN899f16b777754991924d45661d0c60bb.min.js"`

Comment (iamitshri) [reply 24life/scripts/delayed.js:19]: @synox  

- Could you please guide if hostname expected on line 10 is correct?
-  How do we compare dev, stage, and prod?

File: 24life/scripts/delayed.js [hunk 2]
```diff
@@ -1,7 +1,20 @@
...
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/**
+ * Load the launch library applicable to the domain
+ */
+async function loadAdobeLaunch() {
+  const isProd = window.location.hostname === 'https://www.24hourfitness.com/';
```
Comment (synox) [inline 24life/scripts/delayed.js:10]: Does hostname include the protocol and path?

**Comment by meejain** [comment]
Could see the N/W calls for adobedtm scripts getting called, the moment we click on anything on the page, as shown below - 

image

Hence we should be good with the loading of these scripts for any kind of activity on the page and "Clicking" on an element is one of them. However final confirmation will be from the customer whether they are getting required statistics at their end from Adobe Launch.

File: 24life/scripts/delayed.js (hunk 2 above)
Comment (iamitshri) [reply 24life/scripts/delayed.js]: Fixed

File: 24life/scripts/delayed.js (hunk 1 above)
Comment (synox) [reply 24life/scripts/delayed.js:19]: I guess localhost could be considered dev

**Review by synox — APPROVED** [review]

**Pull request merged**